pkg/remote/trans/netpollmux/server_handler.go

```diff
@@ -477,3 +485,12 @@ func getRemoteInfo(ri rpcinfo.RPCInfo, conn net.Conn) (string, net.Addr) {
 	}
 	return ri.From().ServiceName(), rAddr
 }
+
+func getDefaultSvcInfo(svcMap map[string]*serviceinfo.ServiceInfo) *serviceinfo.ServiceInfo {
```

Comment:
there are two definition of this func, you can move it to trans/common.go, reuse it rather than defining it separately